### ISSUE TITLE
Add "recent blog posts" code snippet

### ISSUE THREAD

**Issue by AnilMistry** [GSTT-CSC/csc-test-site#108]
As we're intending to post more blogs as updates to what we're doing. It would be great to be able to easily link each page to relevant blog posts. 

- Create a HTML code snippet for displaying the 4 most recent blog posts of a certain category. 
- All blog posts should also be labelled with at least one category. these could include
  - Quality mansgement system
  - 10X Workshops
  - STP blog
  - Aide/flip/MLops etc
  - Conferences 
  etc. 
- This function should be generic, and added to the "include" folder.
- This code snippet can then be added to the bottom of each layout page, so focussed blog posts

**Comment by hshuaib90** [comment]
I propose solving this by editing the current categories from "Articles" and "Workshops" to the ones you listed above. In addition we would have an all categories tab as the default which would show all of them first.
![image](https://user-images.githubusercontent.com/19840489/218873094-c54ffc37-c526-4caa-a355-bb5711c1c92f.png)

@AnilMistry @dangerdika What do you think?

**Comment by AnilMistry** [comment]
I agree, remove subdirectory and just have type in the frontmatter yaml metadata for filtering as required in the code snippets. These code snippets are easy to make and are general, so can be added at the bottom of each page template.
